ReleaseHistory.md

```diff
@@ -12,6 +12,7 @@
 * BUG: Fix error `ERR997.NoValidAnalysisTargets` when scanning symbolic link files.
 * BUG: Fix `ERR999.UnhandledEngineException: System.IO.FileNotFoundException: Could not find file` when a file name or directory path contains URL-encoded characters.
 * BUG: Fix error `ERR997.NoValidAnalysisTargets` when ambiguous file/directory references are provided to `OrderedFileSpecifier`. Previously, the code required an explicit directory separator to be added to the end of a directory path. Now, the code inspects the file system and assumes that a reference to an existing directory was intended by the user (even without a trailing separator).
+* BUG: Fix `ArgumentException: Illegal characters in path` when analysis target URI contains characters that are illegal in the file system.
```

Comment:
>ArgumentException [](http://example.com/codeflow?start=12&length=17)

This release note doesn't describe the code location where the fix was made.

Reply:
You haven't described the method that raises the exception. This means that customers who have a stack frame in an exception might not easily find your note. `MultithreadedAnalyzeCommandBase.Run` would provide this information. Although you've documented this specific fix, you haven't described the other code locations you changed, which you could add.